ztp/gitops-subscriptions/argocd/example/siteconfig/example-sno.yaml

```diff
@@ -45,6 +45,7 @@ spec:
         bmcCredentialsName:
           name: "example-node1-bmh-secret"
         bootMACAddress: "AA:BB:CC:DD:EE:11"
+        # Use UEFISecureBoot to enable secure boot
```

Comment:
We should maybe add this same comment to all the example SiteConfigs?

Reply:
I tried this config only for SNOs. But since this config ultimately is part of BMH CR I think it's safe to add the comment.